sdk/tvOS/OoyalaTVSkinSDK/Widgets/OOOoyalaTVOptionCell.m

```diff
@@ -0,0 +1,83 @@
+
+//
+//  OOOoyalaTVOptionCell.m
+//  OoyalaTVSkinSDK
+//
+//  Created by Ileana Padilla on 7/19/17.
+//  Copyright © 2017 ooyala. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "OOOoyalaTVOptionCell.h"
+#import "OOOoyalaTVConstants.h"
+
+@implementation OOOoyalaTVOptionCell
+
+- (id)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    
+    if (self) {
+        
+        self.optionLabel = [[UILabel alloc] initWithFrame:CGRectMake(labelWidth, 0, labelWidth * 3, labelHeight * 4)];
+        self.checkedLabel = [[UILabel alloc] initWithFrame:CGRectMake(labelHeight, 0, labelHeight * 2, labelHeight * 4)];
+        self.checkedLabel.text = @"✓";
+        self.checkedLabel.textColor = [UIColor whiteColor];
+        [self.checkedLabel setHidden:YES];
+        self.optionLabel.textColor = [UIColor whiteColor];
+        self.optionLabel.font = [UIFont fontWithName:@"Helvetica Neue" size:42];
+    }
+    self.layer.cornerRadius = cornerRadius;
+    [self addSubview:self.checkedLabel];
+    [self addSubview:self.optionLabel];
+    
+    [self setUserInteractionEnabled:YES];
+    
+    return self;
+}
+
+- (void)didUpdateFocusInContext:(UIFocusUpdateContext *)context withAnimationCoordinator:(UIFocusAnimationCoordinator *)coordinator {
+    [super didUpdateFocusInContext:context withAnimationCoordinator:coordinator];
+    
+    if (self == context.nextFocusedView) {
+        [coordinator addCoordinatedAnimations:^{
+            context.nextFocusedView.transform = CGAffineTransformMakeScale(1.1, 1.1);
+            context.nextFocusedView.backgroundColor = [[UIColor lightGrayColor] colorWithAlphaComponent:0.50];
+        } completion:^{
+            // completion
```

Comment:
since you do nothing on `completion`, can you pass nil instead of an empty block?